### ISSUE TITLE
[Screen Reader – BotKit – Docs]: The Purpose of the link is not defined descriptively.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#3]
**User Experience:**
Users who rely on screen readers will affect to navigate, if the purpose of the links is not defined descriptively.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Screen Reader: Narrator

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Docs” link and select it.
Step 3: Navigate to “npm v0.74” link using tab key.
Step 4: Verify the Purpose of the link is defined descriptively for “npm v0.74” link.
Step 5: Similarly verify “dependencies up to date” and “license MIT” links.

**Actual Result:**
The Purpose of the links is not defined descriptively for the below links.
“npm v0.74”
“dependencies up to date” 
“license MIT” links.

**Expected Result:**
The Purpose of the links should be defined descriptively for the below links.
“npm v0.74”
“dependencies up to date” 
“license MIT” links.

**Note**: Same issue repros “here” link also.

**MAS Reference Link:**
[](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7})
![MAS2 4 4_BotKit_Docs_Link Purpose](https://user-images.githubusercontent.com/48272005/54176450-e0554f00-44b4-11e9-8040-89f51bee22a2.png)
![MAS2 4 4_BotKit_Note issue](https://user-images.githubusercontent.com/48272005/54176451-e0554f00-44b4-11e9-8bd3-320dc4097ac8.png)

**Comment by benbrown** [comment]
@v-gasikk Would adding title tags to these links be an appropriate remediation for this?

**Comment by awalia13** [comment]
Adding aria-label or Title both will work.

**Comment by benbrown** [comment]
Thanks!  This is done.

**Comment by v-gasikk** [comment]
Issue is Fixed. Hence Closing the bug.
Now the Purpose of the link is defined descriptively.
